egui_web/src/backend.rs

```diff
@@ -15,14 +15,30 @@ impl WebBackend {
     pub fn new(canvas_id: &str) -> Result<Self, JsValue> {
         let ctx = egui::CtxRef::default();
 
-        let painter: Box<dyn Painter> =
+        #[cfg(feature = "use_glow_painter")]
+        let painter: Box<dyn Painter> = {
+            let canvas = canvas_element_or_die(canvas_id);
+            let gl_ctx =
+                egui_glow::create_context_for_canvas::init_glow_context_from_canvas(&canvas);
+            let dimension = [canvas.width() as i32, canvas.height() as i32];
+            let painter = egui_glow::Painter::new(&gl_ctx, Some(dimension));
+            Box::new(crate::glow_wrapping::WrappedGlowPainter {
```

Comment:
this would be nicer as a constructor of `WrappedGlowPainter`, so the call here would just be 

`Box::new(crate::WrappedGlowPainter::new(canvas_id))`

Then the fields of `WrappedGlowPainter` could also be private!

Reply:
Ok i fixed it.